packages/lab/src/Flow/stories/Base/Asset.tsx

```diff
@@ -116,36 +120,40 @@ export const Asset: HvFlowNodeFC<NodeGroup, AssetData> = (props) => {
         ]}
         outputs={[
           {
-            label: (
-              <div style={{ display: "flex", gap: 8, alignItems: "center" }}>
-                Sensors
-                <HvButton size="sm" variant="primarySubtle">
-                  Configure
-                </HvButton>
-              </div>
-            ),
+            id: "sensors",
+            label: "Sensors",
+            actions: [
+              {
+                id: "config",
+                label: "Configure",
+              },
+            ],
+            actionsCallback: handleActionCallback,
+            actionsButtonVariant: "primarySubtle",
+            actionsPlacement: "right",
+            // Actions shared by all outputs in the group
+            defaultOutputsActions: {
+              actions: [
+                {
+                  id: "edit",
+                  label: "Edit",
+                  icon: <Edit />,
+                },
+              ],
+              actionsCallback: handleActionCallback,
+              actionsButtonVariant: "primaryGhost",
+              actionsIconButton: true,
+            },
```

Comment:
> [!IMPORTANT]  
> Not a request for changes, as this is the whole point of the story

To me this feels more complicated than the `React.ReactNode` we had before.

Previously, I easily knew exactly what was happening and had full control over it.
Now, I have a new API, with quite a few options to learn about and caveats to be aware of.

---

As a sidenote, one wouldn't have to repeat code for common actions:

```tsx
const outputsWithAction = outputs.map(output => ({
  ...output,
  label: (
    <div className={classes.outputLabel}>
      <IconButton onClick={() => {}}><Edit aria-label="Edit" /></IconButton>
      {label}
    </div>
}))

Reply:
I agree, only having `React.ReactNode` always made more sense to me. It's less confusing and users can do anything they want. There's a lot of properties because I wanted to make it as customisable as possible but now I'm thinking I should have done the opposite. 
Maybe we can assume that if you want to use actions on the labels, they'll always be placed on the same place and have the same style. Users would only able to define the labels and icons (`actions`), and they would have access to the callback. 
Do you think it would be preferable?